string name
int64 id
int64 initial_state
int64 final_state
int64[] trans_
float64 value
